### ISSUE TITLE
Export Multiaddr to allow type checking

### ISSUE THREAD

**Issue by robertkiel** [multiformats/js-multiaddr#106]
https://github.com/multiformats/js-multiaddr/blob/9ffafdb8e52cd0c922b71333f1cb7091f4cbc58b/src/index.d.ts#L33

Right now, only the constructor and the static methods are exported. This limits the usage to sth like

```ts
let x = Multiaddr('...')
```

but it does not allow requiring Multiaddr as e.g. function parameters
```ts
function foo(ma: Multiaddr): void {
    // do sth. nice
}
```

Adding this might help
```ts
export { Multiaddr }
```

**Comment by robertkiel** [comment]
I've already created a simplified declaration file that fits my usage

https://github.com/hoprnet/hopr-types/blob/557f90622ac351b3e43ed4cb2b9ac9ff16930cc4/types/multiaddr.d.ts

**Comment by jacobheun** [comment]
Should be resolved now via #112
